src/mixins/countries-provinces-mixin.ts

```diff
@@ -60,4 +60,17 @@ export default class CountriesProvincesMixin extends Vue {
     window['countryRegionsCache'][code] = result
     return result
   }
+
+  getCanadaRegionsExcludeBC (code: string): Array<object> {
```

Comment:
method similar to `getCountryRegions` just excluding BC province

Reply:
OK, I see that now.

1. It's redundant to pass in `code` since this function is specific to CA (by name anyway).
2. I think you should use a different cache in case getCountryRegions() is called just before this, since line 66 would return the wrong list.

Reply:
Thanks for the fix. I'm OK with this now.